include/AutopinPlus/ControlStrategy.h

```diff
@@ -162,7 +162,15 @@ class ControlStrategy : public QObject {
 	 * \return The new pinning. The tasks will be pinned accordingly
 	 * by changePinning.
 	 */
-	virtual Pinning getPinning(const Pinning &current_pinning) const;
```

Comment:
I need to modify the class variable pinCount in the Scatter implementation of getPinning.